### PR TITLE
add IncredibleJunior plugins

### PR DESCRIPTION
**Pull request by cmeriaux**
Add IncredibleJunior plugins:
- Tortoise SVN
- Solution Hub
- Solution Hub UI
- Solution Tools
- Open File In Solution

### CODE REVIEW

**Comment by cmeriaux** [comment]
The plugin author did not set the dll version. I'll try to contact him.

**Comment by chcg** [comment]
@incrediblejr Could you please check the missing dll versions to be able to publish your plugins via the PluginAdmin list.

**Comment by incrediblejr** [comment]
I'll have look at this when time permits, hopefully this week.

**Comment by MetaChuh** [comment]
@cmeriaux 

what is your motivation to push older plugins from another developer to the official notepad++ list for him ?
please also check one of the ikarus warnings at [virustotal](https://www.virustotal.com/gui/file/cb01ceeca4dc75b4e83cac9e686db83e9f579e65d3a339c7476a43864b957f82/detection) and have a look at the source code, to make sure all is clean.

many thanks and best regards.

**Comment by cmeriaux** [comment]
@MetaChuh  My motivation is clear : I use this plugin and I need it.
As the plugin is not buildable yet and the author didn't answered; the plugin seems to be dead. I moved to another solution based on python script plugin with a custom python script to achieve my needs.

**Comment by incrediblejr** [comment]
I just published a [release](https://github.com/incrediblejr/nppplugins/releases) but unfortunately do not currently have the time to setup and test nppPluginList.

**Comment by cmeriaux** [comment]
@incrediblejr thanks for the new release. I've updated the pluginList json file. unfortunately it doesn't works because your packages are wrongly build. You must remove the first folder level inside your zip. Can you do it please ?
Thanks

**Comment by incrediblejr** [comment]
@cmeriaux Sure, look [here](https://github.com/incrediblejr/nppplugins/releases/tag/v3.0.1)

**Comment by cmeriaux** [comment]
Thanks @incrediblejr now it's working. Appveyor build is now OK. 
I've tested on my computer, it's also OK (x86 and x64).
@chcg PR is ready to be merged.
Thanks

File: src/pl.x86.json [hunk 1]
```diff
@@ -825,7 +880,7 @@
 			"repository": "https://github.com/chcg/NPP_ExportPlugin/releases/download/0.2.9.21/NppExport_0.2.9.21_x86.zip",
 			"description": "NppExport is a true WYSIWYG exporter. It allows you not only to save your source code as a html/rtf file, but also to copy your source code in the clipboard in RTF/HTML format, so you can paste it into your word processor (MS Word, Abiword, openoffice.org Writer) to get the same visual effect.",
 			"author": "",
-			"homepage": "https://github.com/chcg/NPP_ExportPlugin"
+			"homepage": "https://github.com/chcg/NPP_ExportPlugin",
```
Comment (chcg) [inline src/pl.x86.json]: @cmeriaux Thanks for fixing also this.

Comment (cmeriaux) [reply src/pl.x86.json]: you're welcome

**Pull request merged**